packages/@coorpacademy-app-review/src/actions/ui/answers.ts

```diff
@@ -52,12 +55,18 @@ export const editAnswer =
   (answer: string[]) =>
   (dispatch: Dispatch, getState: () => StoreState): EditAnswerAction => {
     const state = getState();
-    const currentSlideRef = get(['ui', 'currentSlideRef'])(state);
-    const userAnswers = get(['ui', 'answers'])(state);
-    const slide = get(['data', 'slides', currentSlideRef])(state);
-    const questionType = get(['question', 'type'])(slide);
+    const currentSlideRef = get(['ui', 'currentSlideRef'], state);
+    const userAnswers = get(['ui', 'answers'], state);
+    const slide = get(['data', 'slides', currentSlideRef], state);
+    const questionType = get(['question', 'type'], slide);
+
+    if (!slide || !questionType)
+      throw new Error('No slide was found or questionType was not found');
+
     const type = get(questionType, ANSWER_EDIT);
 
+    if (!type) throw new Error('questionType is not supported');
```

Comment:
pareil, avec le code que j'ai te mis, ça n'arrive jamais selon TS

Reply:
oui (pour le typing) mais si jamais il y a un type different, pas null mais un nouveau type de question là on a besoin de ça car `ANSWER_EDIT` n'aura pas le prop

Reply:
Ok... bon, mais du coup change le message d'erreur
'Question type ${questionType} is not supported'